integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/InitializationTasks.java

```diff
@@ -335,7 +336,7 @@ public void close() {
     if (SKIP_CLEANUP) {
       logger.info("Skipping RESULTS_ROOT clean up after test execution");
     } else {
-      if (!OKD) {
+      if (!OKD && !OKE_CLUSTER) {
```

Comment:
Why are we skipping istio un-nstallation on OKE ?

Reply:
it causes hanging the shell script after adding that option --set components.cni.enabled=true, OKD also skip uninstall